lib/main.gi

```diff
@@ -2713,3 +2713,32 @@ end);
 ##
 #E
 ##
+
+IsMaximal:=function(N,A,D)
+The function IsMaximal checks whether A/D is maximal abelian subgroup of N/D 
+
+----------------------------
+
+IsMaximal:=function(N,A,D)

```

Comment:
I suggest to have more specific name, e.g. `IsMaximalAbelianInFactorGroup`. Otherwise there is a risk of mane clashes with some other code, or user accidentally misunderstanding its meaning.